app/views/client/power_tab.php

```diff
@@ -1,66 +1,126 @@
-					<tr>
-					<td>Charge Remaining (mAh):</td>
-					<td><?=$power->current_capacity?></td>
+				<tr>
+					<th data-i18n="power.current_capacity"></th>
+					<td id="power-current_capacity"></td>
 				</tr>
-					<tr>
-					<td>Charged %:</td>
-					<td><?=$power->current_percent?></td>
+				<tr>
+					<th data-i18n="power.current_percent"></th>
+					<td id="power-current_percent"></td>
 				</tr>
-					<tr>
-					<td>Cycle Count:</td>
-					<td><?=$power->cycle_count?></td>
+				<tr>
+					<th data-i18n="power.cycle_count"></th>
+					<td id="power-cycle_count"></td>
 				</tr>
-					<tr>
-					<td>Temperature:</td>
-					<td><?php
-						if ($power->temperature == 0 || $power->temperature == ""){
-							echo "";
-							}
-						else {
-							if (conf('temperature_unit') == "F"){
-								// Fahrenheit
-								echo round(((($power->temperature * 9/5 ) + 3200 ) / 100), 2) . "° F";
-								}
-							else {
-								// Celsius
-								echo $power->temperature = ($power->temperature / 100) . "° C";
-								}
-						}
-						?></td>
+				<tr>
+					<th data-i18n="power.condition"></th>
+					<td id="power-condition"></td>
 				</tr>
-					<tr>
-					<td>Condition:</td>
-					<td><?=$power->condition?></td>
+				<tr>
+					<th data-i18n="power.temperature"></th>
+					<td id="power-temperature"></td>
 				</tr>
-			</tbody>
-		</table>
+			</table>
+		</div>
+		<div class="col-md-6">
+			<div style="height: 512px" id="map-canvas"></div>
+		</div>
+	</div>
+
+<script>
+$(document).on('appReady', function(e, lang) {
+	
+	// Get directory_service data
+	$.getJSON( appUrl + '/module/power/get_data/' + serialNumber, function( data ) {
+		if( ! data.manufacture_date){
+			$('#power-msg').text(i18n.t('no_data'));
+		}
+		else{
+			
+			// Hide
+			$('#power-msg').text('');
+			$('#power-view').removeClass('hide');
+
+            // Add strings
+			$('#power-design_capacity').text(data.design_capacity);
+			$('#power-max_capacity').text(data.max_capacity);
+			$('#power-current_capacity').text(data.current_capacity);
+			$('#power-cycle_count').text(data.cycle_count);
+
+            // Format Manufacture date
+            if(data.manufacture_date === '1980-00-00'){
+	        		$('#power-manufacture_date').addClass('danger').html(i18n.t('widget.power.now'));
+            } else {
+	        	if(data.manufacture_date){
+	        		a = moment(data.manufacture_date)
+	        		b = a.diff(moment(), 'years', true)
+	        		if(a.diff(moment(), 'years', true) < -4)
+	        		{
+	        			$('#power-manufacture_date').addClass('danger')
+	        		}
+	        		if(Math.round(b) == 4)
+	        		{
+	        			
+	        		}
+	        		$('#power-manufacture_date').html('<span title="'+data.manufacture_date+'">'+moment(data.manufacture_date).fromNow());
+	        	}
+            }
+            
+            // Format battery condition
+            data.condition = data.condition == 'Normal' ? '<span class="label label-success">Normal</span>' : 
```

Comment:
While you're on it, you may want to localise the battery condition messages as well! :-)